src/cli.rs

```diff
@@ -172,6 +172,12 @@ pub fn cli() -> App<'static, 'static> {
      //         .long("show-time-diff")
      //         .conflicts_with("output")
      //         .help( "Show the time difference between the occurence of equal tags in terminal output"))
+        .arg(Arg::with_name("dev")
```

Comment:
can we get this in a seperate PR?

Reply:
your request is my command... I'll change it immediately